plugins/deep-link/src/lib.rs

```diff
@@ -154,23 +166,55 @@ mod imp {
 
     /// Access to the deep-link APIs.
     pub struct DeepLink<R: Runtime> {
-        #[allow(dead_code)]
         pub(crate) app: AppHandle<R>,
-        #[allow(dead_code)]
         pub(crate) current: Mutex<Option<Vec<url::Url>>>,
+        pub(crate) config: Option<crate::config::Config>,
     }
 
     impl<R: Runtime> DeepLink<R> {
+        #[cfg(desktop)]
+        #[doc(hidden)]
+        pub fn handle_cli_arguments<S: AsRef<str>, I: Iterator<Item = S>>(&self, mut args: I) {
```

Comment:
if this is used by tauri-plugin-single-instance, maybe we should document it instead

Reply:
it is internal to our workspace yes, but if it is used in another plugin, it should be documented and potentiolly allow other users to add same integration